src/main/scala/mesosphere/marathon/event/Events.scala

```diff
@@ -96,6 +93,16 @@ case class Unsubscribe(
   timestamp: String = Timestamp.now().toString)
     extends MarathonSubscriptionEvent
 
+case class EventStreamAttached(
+  remoteAddress: String,
+  eventType: String = "event_stream_attached",
+  timestamp: String = Timestamp.now().toString) extends MarathonSubscriptionEvent
+
+case class EventStreamDetached(
+  remoteAddress: String,
+  eventType: String = "event_stream_detached",
+  timestamp: String = Timestamp.now().toString) extends MarathonSubscriptionEvent
```

Comment:
I do not particularly like that you use String for `timestamp` but it doesn't really matter that much either.